CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
 #============================================================================
 # Set up the project
 #============================================================================
-ign_configure_project(VERSION_SUFFIX pre1)
+ign_configure_project(VERSION_SUFFIX)
```

Comment:
We should keep the `pre1` on `main` so the nightlies are versioned correctly. We only remove this when we're ready for a stable release.

Reply:
fixed. 07555cc